LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015-present Rapptz
+Copyright (c) 2021-present tag-epic
```

Comment:
I am not sure if you are allowed to take him out commpletly. Most of the code is still from his repo

Reply:
Danny's line wasn't removed here????

Reply:
Hasn't been taken out. Epic just added

Reply:
my bad. I should switch to my pc, reviewing from the phone is hard. sry